mozdownload/scraper.py

```diff
@@ -611,7 +613,7 @@ def binary_regex(self):
                  'win64': r'^%(APP)s(%(STUB_NEW)s|(?:\sSetup\s|-)%(STUB)s%(VERSION)s)\.%(EXT)s$',
                  }
         return regex[self.platform] % {
-            'APP': self.application,
+            'APP': APPLICATIONS_TO_BINARY_NAME.get(self.application, self.application),
```

Comment:
While thinking more about this line, I would propose that we immediately update all those occurrences in `binary_regex` across this file. It will make it more consistent.

Reply:
Hey Adam. I think you missed this particular comment? It would be great to get this changed also for the `binary_regex` property for Daily, Tinderbox, and Try scrapers. Thanks!

Reply:
I simply had to leave for "boofen" and didn't manage to do it before. In addition I changed the APP to BINARY_NAME for further consistency.

Reply:
Ah cool! :) Great to see that this change went in with your most recent commit.

Thanks for also changing the placeholder from `APP` to `BINARY_NAME`. I think that makes sense now.

Reply:
Ah cool! :) Great to see that this change went in with your most recent commit.

Thanks for also changing the placeholder from `APP` to `BINARY_NAME`. I think that makes sense now.

Reply:
Ah cool! :) Great to see that this change went in with your most recent commit.

Thanks for also changing the placeholder from `APP` to `BINARY_NAME`. I think that makes sense now.